--- dmraid/lib/format/ataraid/pdc.c~	2005-03-01 15:29:05.000000000 +0100
+++ dmraid/lib/format/ataraid/pdc.c	2005-04-17 14:09:30.000000000 +0200
@@ -102,7 +102,7 @@
 	CVT32(pdc->raid.magic_1);
 
 	for (disk = pdc->raid.disk;
-	     disk < &pdc->raid.disk[pdc->raid.total_disks;
+	     disk < &pdc->raid.disk[pdc->raid.total_disks];
 	     disk++) {
 		CVT32(disk->magic_0);
 		CVT32(disk->disk_number);
